docs/_basic/ja_listening_events.md

```diff
@@ -15,16 +15,16 @@ order: 3
 const welcomeChannelId = 'C12345';
 
 // 新しいユーザーがワークスペースに加入したタイミングで、指定のチャンネルにメッセージを送信して自己紹介を促す
-app.event('team_join', async ({ event, client }) => {
+app.event('team_join', async ({ event, client, logger }) => {
   try {
     const result = await client.chat.postMessage({
       channel: welcomeChannelId,
-      text: `Welcome to the team, <@${event.user.id}>! 🎉 You can introduce yourself in this channel.`
```

Comment:
I don't think this change is correct.

Reply:
Whoops, I think you're right, I was testing with the wrong event!  I was going to check the doc but there isn't a full example there either: https://api.slack.com/events/team_join.

Reply:
Indeed, the document should be comprehensive but you can rely on the outputs by the Java SDK project at this moment:

* https://github.com/slackapi/java-slack-sdk/blob/main/json-logs/samples/events/TeamJoinPayload.json
* https://github.com/slackapi/java-slack-sdk/blob/main/slack-api-model/src/main/java/com/slack/api/model/event/TeamJoinEvent.java

Reply:
Thanks! I just tested it again to be sure and reverted the change now.